packages/apps/transfer/src/pages/transfer/module-explorer/index.tsx

```diff
@@ -47,8 +47,7 @@ const GetCode: FC = () => {
       const data = await describeModule(
         moduleName,
         convertIntToChainId(moduleChain),
-        chainNetwork[network].network,
-        chainNetwork[network].server,
+        network,
         'not-real',
```

Comment:
Maybe this could also be a constant. For context, when we send the requests for the module explorer we set the sender as 'not-real'.

Reply:
Solved 👍